lib/rufo/command.rb

```diff
@@ -71,13 +72,37 @@ def format_args(args)
       syntax_error |= result == CODE_ERROR
     end
 
+    STDERR.puts squiggly_heredoc_warning unless @squiggly_warning_files.empty?
+
     case
     when syntax_error then CODE_ERROR
     when changed      then CODE_CHANGE
     else                   CODE_OK
     end
   end
 
+  def squiggly_heredoc_warning
+    <<-EOF
+Rufo Warning!
+  File#{squiggly_pluralize} #{squiggly_warning_files} #{squiggly_pluralize(:has)} not been formatted due to a problem with Ruby version #{RUBY_VERSION}
```

Comment:
I wonder if this should include a reference to the problem? That way someone could avoid using squiggly heredocs rather than upgrading if that is a challenge.

Reply:
Good point - I've added a reference to a wiki page.